js/clone.js

```diff
@@ -156,8 +156,14 @@
 	 * @param $container .rwmb-input container
 	 */
 	function toggleRemoveButtons( $container ) {
-		var $clones = $container.children( '.rwmb-clone' );
-		$clones.children( '.remove-clone' ).toggle( $clones.length > 1 );
+
+		var $clones = $container.children( '.rwmb-clone' ),
+		    minClone = 1;
+	
+		if( $container.data( 'min-clone' ) ){
```

Comment:
Can you please add a space after `if` and before `{` to match the coding standard. Sorry to comment on this twice.

Reply:
very sorry this is what it's come to. Is there a book on this stuff that I can ...  check out from the library, or something?

Reply:
Yes, here it is: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/

For PHP, please follow this: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/

We don't strictly follow **all** rules, but we do follow for all spacing rules.

Reply:
Hmm, Github doesn't notify me when you replied :(